src/components/shared/Button/Close.js

```diff
@@ -10,16 +10,18 @@ type CloseButtonType = {
 };
 
 function CloseButton({ handleClick, buttonClass, tooltip }: CloseButtonType) {
-  return dom.div(
-    {
-      className: buttonClass ? `close-btn ${buttonClass}` : "close-btn",
-      onClick: handleClick,
-      title: tooltip
-    },
-    Svg("close")
+  return (
+    <div
+      className={buttonClass ? `close-btn ${buttonClass}` : "close-btn"}
+      onClick={handleClick}
+      title={tooltip}
+    >
+      {Svg("close")}
```

Comment:
Unrelated but it'd be nice to update `Svg` to take a `name` prop and then we could have these calls be JSX too.